src/pages/[id].page.jsx

```diff
@@ -29,20 +29,33 @@ export default function Details({ details }) {
   const [htmlCode, setHtmlCode] = useState('');
   const [cssCode, setCssCode] = useState('');
   const [javascriptCode, setJavascriptCode] = useState('');
+  const [canRenderSandpack, setCanRenderSandpack] = useState(false);
 
   // dynamic imports need to be asynchronous
   useEffect(() => {
-    const updateCodeBlocks = async () => {
-      const html = await import(`../code-examples/${slug}/html.js`);
-      setHtmlCode(html.default);
+    const updateSandpackExamples = async () => {
+      try {
+        const html = await import(`../code-examples/${slug}/html.js`);
+        setHtmlCode(html.default);
 
-      const css = await import(`../code-examples/${slug}/css.js`);
-      setCssCode(css.default);
+        const css = await import(`../code-examples/${slug}/css.js`);
+        setCssCode(css.default);
 
-      const javascript = await import(`../code-examples/${slug}/javascript.js`);
-      setJavascriptCode(javascript.default);
+        const javascript = await import(
+          `../code-examples/${slug}/javascript.js`
+        );
+        setJavascriptCode(javascript.default);
+        setCanRenderSandpack(true);
+        return;
```

Comment:
This `return` statement isn't necessary, we can get rid of it.

Reply:
I was getting some weird linting errors. But maybe if I delete the if statement in the catch block, I won't!